code/controllers/subsystem/modular_computers.dm

```diff
@@ -35,22 +38,44 @@ SUBSYSTEM_DEF(modular_computers)
 		if(prog.available_on_syndinet)
 			available_antag_software.Add(prog)
 
-///Checks if at least one ntnet relay is functional.
-/datum/controller/subsystem/modular_computers/proc/check_relay_operation()
-	for(var/obj/machinery/ntnet_relay/relays as anything in ntnet_relays)
-		if(!relays.is_operational)
-			continue
-		return TRUE
-	return FALSE
-
 ///Attempts to find a new file through searching the available stores with its name.
 /datum/controller/subsystem/modular_computers/proc/find_ntnet_file_by_name(filename)
 	for(var/datum/computer_file/program/programs as anything in available_station_software + available_antag_software)
 		if(filename == programs.filename)
 			return programs
+	return null
 
 ///Attempts to find a chatorom using the ID of the channel.
 /datum/controller/subsystem/modular_computers/proc/get_chat_channel_by_id(id)
 	for(var/datum/ntnet_conversation/chan as anything in chat_channels)
 		if(chan.id == id)
 			return chan
+	return null
+
+/**
+ * Records a message into the station logging system for the network
+ * Arguments:
+ * * log_string - The message being logged
+ */
+/datum/controller/subsystem/modular_computers/proc/add_log(log_string)
+	set waitfor = FALSE // so process keeps running
```

Comment:
what sleeps here?

Reply:
This was copied over from the network subsystem, I havent actually looked into it much

Reply:
I'm removing it because modular computers don't even process lol, this seems pretty unecessary. It was probably for when it was on the networks subsystem which was adding logs all the time and processed.

Reply:
~~what sleeps here tho did it ever actually do anything~~

Reply:
It doesn't, it might've been a leftover from a previous version that was removed